client/src/components/App.jsx

```diff
@@ -15,25 +14,34 @@ class App extends Component {
       minNightStay: 0,
       cleaningFee: 0,
       addtlGuestFee: 0,
+      guestDropDownActive: false,
     };
+    this.handleGuestDropDown = this.handleGuestDropDown.bind(this);
     this.postBooking = this.postBooking.bind(this);
   }
-  componentDidMount () {
-    this.fetchDetailsAndAvailNights(ROOM);
+  componentDidMount() {
+    this.fetchDetailsAndAvailNights(window.location.pathname.split('/')[2]);
   }
-  fetchDetailsAndAvailNights (roomId) {
-    axios.get(`/rooms/${roomId}`)
+  fetchDetailsAndAvailNights(roomId) {
+    axios.get(`/reservations/${roomId}`)
       .then(db => this.updateState(db.data))
-      .catch(err => console.error(err));
+      .catch(err => err);
+  }
+  handleGuestDropDown(e) {
+    e.preventDefault();
+    this.setState(prevState => ({ guestDropDownActive: !prevState.guestDropDownActive }));
   }
   postBooking () {
 
   }
-  updateState (data) {
+  updateState(data) {
     const { avg_rating, total_ratings, max_guests, min_night_stay, cleaning_fee, addtl_guest_fee } = data[0][0];
-    const avgNightlyRate = data[1].reduce((acc, night) => acc = night.rate > acc ? night.rate : acc, 0);
+    const avgNightlyRate = data[1].reduce((acc, night) => {
+      acc = night.rate > acc ? night.rate : acc;
```

Comment:
I like this ternary format.  I might have to try this out if it's ever applicable.

Reply:
Thanks Brian!